### DISCUSSION TITLE
screen lights flash randomly

### DISCUSSION THREAD

**Discussion by lolo020678** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#626] [Q&A]
Hello,

I have a pump S100101 (Square) and SPA Bahamas. I have PCB version B. The nodemcu V3 works fine and the connection to wifi is ok. The problem is that the led in the screen flash randomly. I have use the level converter blue. Do you think the problem is the level converter ? I need used the red one ?

Thank you for your response.

**Comment by visualapproach** [comment]
hello, you probably need to use the B ports with filters installed. Possibly the capacitors can be omitted, but you have to try.

**Comment by lolo020678** [comment]
Hello. I have received today the level converter (the red one) and now all work fine.

I confirmed for the SPA Bahamas the system works.
